lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc

```diff
@@ -624,8 +624,8 @@ function dosomething_campaign_get_scholarship($nid) {
  * @return string
  *  Formatted time (e.g. "4 days left")
  */
-function dosomething_campaign_get_hot_time_left(&$vars) {
-  $today = $vars['hm_today'];
+function dosomething_campaign_get_hot_time_left($node, &$vars = NULL) {
```

Comment:
`$node` is used [here](https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_campaign/dosomething_campaign_hot_shares/dosomething_campaign_hot_shares.cron.inc#L30)